dp_consistency/src/noiseninja/solver.py

```diff
@@ -6,7 +6,7 @@
 
 from src.noiseninja.noised_measurements import SetMeasurementsSpec
 
-SOLVER = "clarabel"
+SOLVER = "highs"
```

Comment:
Without this change, this code is broken at head :( 
Lets keep this.

Reply:
Could you make this a flag of the reporting tool?

Doesn't have to be on this PR.